src/Support/Testing/Fakes/ConsumerFake.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Junges\Kafka\Support\Testing\Fakes;
+
+use Closure;
+use Junges\Kafka\Config\Config;
+use Junges\Kafka\Contracts\KafkaConsumerMessage;
+use Junges\Kafka\Contracts\MessageDeserializer;
+use Junges\Kafka\MessageCounter;
+use RdKafka\Conf;
+
+class ConsumerFake
+{
+    private MessageCounter $messageCounter;
+    private bool $stopRequested = false;
+    private ?Closure $onStopConsume = null;
```

Comment:
I think we can also use PHP property promotion for this two variables. There is some other places where I need to fix it but no need to do it on this PR, just fix this one and it's fine.

Reply:
You mean this?

```php
public function __construct(
    private Config $config,
    private array $messages = [],
    private bool $stopRequested = false,
    private ?Closure $onStopConsume = null
) {
}
```

Reply:
yep